frontend/views/components/GroupSettings/InviteActionButton.vue

```diff
@@ -0,0 +1,60 @@
+<template lang="pug">
+menu-parent
+  menu-trigger.is-icon
+    i.icon-ellipsis-v
+
+  menu-content.c-content
+    ul
+      menu-item(
+        tag='router-link'
+        item-id='hash-3'
+        to='/somewhere'
+      )
+        i18n.icon-vote-yea.c-icon(tag='i') See original proposal
+      menu-item(
+        tag='router-link'
+        item-id='hash-3'
+        v-if='!linkExpired'
```

Comment:
it's a good habit to keep v-if as the first attribute
https://vuejs.org/v2/style-guide/#Element-attribute-order-recommended

Reply:
Thanks for the advice.